include/treelite/tree.h

```diff
@@ -525,7 +555,7 @@ class ModelImpl : public Model {
   inline size_t GetNumTree() const override {
     return trees.size();
   }
-  void SetTreeLimit(size_t limit) override {
+  inline void SetTreeLimit(size_t limit) override {
```

Comment:
`inline` here is redundant, as a function defined inside a `class`/`struct` definition is implicitly inline.

Reply:
Fixed in the latest commit.